### ISSUE TITLE
Add a List with command-overview to pop in

### ISSUE THREAD

**Issue by GeorgJuranek** [GeorgJuranek/capstone-project#10]
## Value proposition

**As a** user,
**I want to** have an overview on my control-commands
**so that** i can easily look them up

## Description

<img src="https://user-images.githubusercontent.com/104508413/177791092-56cae17a-9324-4d83-a48c-80efa36491c9.png"
width=720px>

## Acceptance criteria

- [ ] When button is clicked a card comes in from out-of-screen bottom direction and can be toggled backwards
- [ ] I can tap on the commands on the left, to get info-text on the right side of card
- [ ] Other Elements cant be interacted with, while card is extended
- [ ] Card comes and goes with a little rush-in animation

## Complexity

Small

## Tasks

- [ ] Create branch
- [ ] Create Button 
- [ ] Create MagicCard-Component
- [ ] MagicCard contains a List and a section for text
- [ ] text in section changes when click on other List-item
- [ ] activated MagicCard is always above other elements
- [ ] Write test for MagicCard
- [ ] Write story for MagicCard

**Comment by ahohnsen** [comment]
Hey Georg,

Nicely done. A few suggestions:

**Comment by ahohnsen** [comment]
Hi Georg,

Nicely done! :) I have a few suggestions:

- I would suggest rephrasing the value proposition as line 2 & 3 are almost the same... e.g.:

_As a user,
I want to have an overview of my control-commands
so that I do not need to remember them_

- I am not quite sure what the following means: _"... and can be toggled backwards"_ - Do you mean that with another click on the button will hide the MagicCard again? If so, looking at your scribble, I am wondering if only the little grey triangle will be the button to open the MagicCard. 🤔 Maybe you can rephrase it to make it a bit more clear.

**Comment by GeorgJuranek** [comment]
Hello,
Thank you for your feedback: i tried to be more specific about the buttons in AcCr-text and added arrows for the toggle-back options in description-image. Hope this make it more understandable.
